personalcapital/personalcapital.js

```diff
@@ -207,6 +208,38 @@ class PersonalCapital {
       accounts, startDate, endDate, intervalType, types));
     return res;
   }
+
+  // accountName must match name on account exactly
+  async updateBalance(accountName /*String*/, newBalance /*Number*/) {
+    const account = await this.getAccountByName(accountName);
+    const updateData = {
+      ...account,
+      isTransferPending: false,
+      isTransferEligible: true,
+      ownershipType: null,
+      employerMatchLimitType: "dollar",
+      requestSource: "USER",
+      balance: newBalance,
+      currentBalance: newBalance,
+      availableBalance: newBalance
+    };
```

Comment:
Payloads should be defined in payload.endpoint as a function. Also ...account seems a bit extreme, does this actually require the whole account object or just the accountid?